controllers/configurationpolicy_controller.go

```diff
@@ -1011,6 +1011,22 @@ func (r *ConfigurationPolicyReconciler) handleObjectTemplates(plc policyv1.Confi
 
 				// Otherwise, set the resolved data for use in further processing
 				plc.Spec.ObjectTemplates[i].ObjectDefinition.Raw = resolvedTemplate.ResolvedJSON
+			} else if isRawObjTemplate {
+				// no templates to resolve, but the template resolver will handle YAML conversion
+				resolvedTemplate, tplErr := tmplResolver.ResolveTemplate(rawData, nil)
+				if tplErr != nil {
+					addTemplateErrorViolation("Error parsing raw template", err.Error())
+				}
+				err := json.Unmarshal(resolvedTemplate.ResolvedJSON, &objTemps)
+				if err != nil {
+					addTemplateErrorViolation("Error unmarshalling raw template", err.Error())
+
+					return
+				}
+
+				plc.Spec.ObjectTemplates = objTemps
+
+				break
```

Comment:
This looks like the same flow as the template logic above. Should the `if` above add an `|| isRawObjTemplate` instead or is there a reason to have this separate?

Reply:
the flow is a bit different now after moving the yaml marshal into the controller